lib/email/receiver.rb

```diff
@@ -1067,7 +1068,9 @@ def forwarded_email_create_topic(
         )
       elsif destination.is_a?(Category)
         return false if user.staged? && !destination.email_in_allow_strangers
-        return false if !user.has_trust_level?(SiteSetting.email_in_min_trust)
+        if user.groups.any? && !user.in_any_groups?(SiteSetting.email_in_allowed_groups_map)
```

Comment:
Shouldn't need `user.groups.any?` here -- if they aren't in any groups then `in_any_groups?` should return false.

Reply:
ya, same as above ^. That code is basically saying "if this users is not staged, is an actual user, and is not part of these groups".